src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -134,9 +134,7 @@ public void initialize() {
                                 group.allSelectedEntriesMatchedProperty());
                     }
                     Text text = new Text();
-                    if (preferencesService.getDisplayGroupCount()) {
-                        text.textProperty().bind(group.getHits().asString());
-                    }
+                    text.textProperty().bind(group.getHits().asString());
```

Comment:
Please leave this if check here, otherwise every group will have a `0` here.

Reply:
Good day,
OK, adjusted in code. Will commit today, when we find solution about PreferencesService in GroupNodeViewModel (see comment from calixtus).